insert_by_period/macros/insert_by_period_materialization.sql

```diff
@@ -97,7 +97,12 @@
       );
     {%- endcall %}
     {% set result = load_result('main-' ~ i) %}
-    {% if 'response' in result.keys() %} {# added in v0.19.0 #}
+    
+    {% if target.type == 'databricks' %} {# databricks provides rows affected in 'data' and not 'response' #}
```

Comment:
We should dispatch this in a macro no, instead of using `target.type == ...`?

Reply:
added a `get_rows_inserted.sql` macro